src/azure-cli/azure/cli/command_modules/appconfig/_params.py

```diff
@@ -0,0 +1,148 @@
+# --------------------------------------------------------------------------------------------
+# Copyright (c) Microsoft Corporation. All rights reserved.
+# Licensed under the MIT License. See License.txt in the project root for license information.
+# --------------------------------------------------------------------------------------------
+
+# pylint: disable=line-too-long
+# pylint: disable=too-many-statements
+
+from azure.cli.core.commands.parameters import (get_enum_type,
+                                                get_location_type, tags_type)
+
+from azure.cli.core.commands.validators import \
+    get_default_location_from_resource_group
+
+from ._validators import (validate_appservice_name_or_id,
+                          validate_connection_string, validate_datetime,
+                          validate_export, validate_import,
+                          validate_import_depth, validate_query_fields,
+                          validate_separator)
+
+
+def load_arguments(self, _):
+    with self.argument_context('appconfig create') as c:
+        c.argument('resource_group_name', options_list=['--resource-group', '-g'], required=True)
```

Comment:
This should use the global alias for resource group already in the CLI.

Reply:
Done.